packages/twenty-front/src/modules/apollo/optimistic-effect/utils/triggerUpdateRelationsOptimisticEffect.ts

```diff
@@ -72,15 +76,15 @@ export const triggerUpdateRelationsOptimisticEffect = ({
         | RecordGqlNode
         | null = updatedSourceRecord?.[fieldMetadataItemOnSourceRecord.name];
 
-      if (
-        isDeeplyEqual(
-          currentFieldValueOnSourceRecord,
-          updatedFieldValueOnSourceRecord,
-          { strict: true },
-        )
-      ) {
+      const noDiff = isDeeplyEqual(
+        currentFieldValueOnSourceRecord,
+        updatedFieldValueOnSourceRecord,
+        { strict: true },
+      );
+      if (noDiff && !isDeletion) {
```

Comment:
shouldn't we have a diff if isDeletion?

However this is not an issue, your approach LGTM too

Reply:
Yep you're right